src/lib/quoteEngine.js

```diff
@@ -256,7 +289,11 @@ const quoteEngine = (store, productId, amount, period, coverAsset) => {
     };
   });
 
-  const allocations = calculateOptimalPoolAllocation(amountToAllocate, poolsData, product.useFixedPrice);
+  // Use a custom pool allocation priority for DeltaPrime (UnoRe)
+  const allocations =
+    productId === DELTA_PRIME_UNO_RE_PRODUCT_ID
```

Comment:
We should maybe add property in store something like 
```
productPriorityPools: {[productId]: [...prorityPoolOrder]}
```
 just pass them from there. So next product just expand the object. Also don't forget to add them to the cache file so they don't get overriden

Reply:
good idea 👍